phoenix-core/src/main/java/org/apache/phoenix/expression/ComparisonExpression.java

```diff
@@ -367,6 +368,13 @@ public void write(DataOutput output) throws IOException {
 
     @Override
     public final <T> T accept(ExpressionVisitor<T> visitor) {
+//        if (visitor instanceof WhereOptimizer.KeyExpressionVisitor) {
```

Comment:
Remove dead comments.

Reply:
Actually, this is not a dead comment. I just found that we need to have this additional check for the RVC comparison

Reply:
Can we add a UT for this then?